scripts/ci/ci_run_airflow_testing.sh

```diff
@@ -149,3 +149,34 @@ do
     exit ${EXIT_CODE}
 done
 set -u
+
+#TODO FIXME
+if [[ "${INTEGRATIONS[*]}" == *"kubernetes"* ]]; then
```

Comment:
hmm @potiuk ? I get `./breeze: line 382: INTEGRATIONS[*]: unbound variable` when starting `./breeze`

Reply:
Yep, Mac OS thingie. I used to catch it early before I switched to Linux. Fixed in https://github.com/apache/airflow/pull/9135

Reply:
Thanks @potiuk !